### DISCUSSION TITLE
I updated to the latest version, Custom Endpoint did not take effect

### DISCUSSION THREAD

**Discussion by sunsky89757** [danny-avila/LibreChat#3623] [Troubleshooting]
### What happened?

[Bug]: I updated to the latest version, Custom Endpoint did not take effect

### Steps to Reproduce

[Bug]: I updated to the latest version, Custom Endpoint did not take effect

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Check your startup container logs, or the ./logs folder at project root. There was a breaking change with speech config
